src/blocks/blocks/icon-list/inspector.js

```diff
@@ -98,6 +99,18 @@ const Inspector = ({
 					max={ 60 }
 				/>
 
+				<SelectControl
```

Comment:
We can move it to the Toolbar instead like the Paragraph.

<img width="472" alt="Screenshot 2021-11-06 at 1 36 19 AM" src="https://user-images.githubusercontent.com/2649903/140572028-95039d2a-21f5-4de0-9a62-7555856de604.png">